### PR TITLE
pronouns for beach patrols

### PR DESCRIPTION
**Pull request by capillarys**
this isn't all done yet but i thought i'd pr them in first, to avoid Even More merge conflicts :sob:

### CODE REVIEW

File: resources/dicts/patrols/beach/border/leaf-bare.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
         "s_c, upon seeing several kits trapped by a Twoleg contraption, immediately jumps into the cold harsh water of the sea. {PRONOUN/s_c/subject/CAP} {VERB/s_c/use/uses} a piece of wood as a makeshift raft and {VERB/s_c/paddle/paddles} the kits back to shore where the rest of the patrol can keep them warm, before racing them back to camp to be cuddled into the pelt of a loving queen."
         ],
         "fail_text": [
-        "To the cat's horror they were too late, the bodies of several kits washing ashore. They mutter a prayer before gently picking up the kits, the patrol swims the little bodies back out into the cold ocean, ensuring that the kits rest within the ocean's embrace."
+        "To the cats' horror, they were too late, the bodies of several kits washing ashore. They mutter a prayer before gently picking up the kits, the patrol swims the little bodies back out into the cold ocean, ensuring that the kits rest within the ocean's embrace."
```
Comment (scribblecrumb) [inline resources/dicts/patrols/beach/border/leaf-bare.json:50]: missed a couple "they"s

File: resources/dicts/patrols/beach/hunting/any.json [hunk 2]
```diff
@@ -120,113 +92,55 @@
...
-        "fail_trait": [
-            "adventurous",
-            "bold",
-            "charismatic",
-            "childish",
-            "confident",
-            "daring",
-            "playful",
-            "righteous"
+            "s_c is too hasty, bouncing after first one prey, then another, and it ends with {PRONOUN/app1/object} empty pawed and exhausted. {PRONOUN/app1/poss/CAP} mentor is disappointed - {PRONOUN/app1/subject} clearly {VERB/app1/need/needs} more training.",
```
Comment (scribblecrumb) [inline resources/dicts/patrols/beach/hunting/any.json]: should be tagging 's_c' in the PRONOUN tags, for consistency

**Review by scribblecrumb — CHANGES_REQUESTED** [review]
was originally gonna read this thoroughly, but then i got the hunting jsons and gave up lmao.  still a couple things i caught on my skim though

File: resources/dicts/patrols/beach/border/leaf-bare.json (hunk 1 above)
Comment (capillarys) [reply resources/dicts/patrols/beach/border/leaf-bare.json:50]: oops lemme get on that!

Comment (capillarys) [reply resources/dicts/patrols/beach/border/leaf-bare.json:50]: wait no i think they're meant to be "theys" bc they're an unknown number of kits?

File: resources/dicts/patrols/beach/hunting/any.json (hunk 2 above)
Comment (capillarys) [reply resources/dicts/patrols/beach/hunting/any.json]: OOPS gotcha!

File: resources/dicts/patrols/beach/border/leaf-bare.json (hunk 1 above)
Comment (capillarys) [reply resources/dicts/patrols/beach/border/leaf-bare.json:50]: and also i interpreted the "they" in the fail text as the cats (so multiple cats) :P but idm changing if it's meant to be one specific cat!

Comment (scribblecrumb) [reply resources/dicts/patrols/beach/border/leaf-bare.json:50]: oop no that's a good point!

**Pull request merged**